### PR TITLE
feat(rating): add aria-label attribute and rating config

### LINKED ISSUE #5579: Rating component missing aria-label
**Bug description:**
The rating component is missing an `aria-label` attribute on the `span` with `role="slider"`. This was discovered using the axe Chrome extension. More information about the issue can be found [here](https://dequeuniversity.com/rules/axe/3.4/aria-input-field-name?application=AxeChrome).

**Example:**
Inspecting the [demo](https://valor-software.com/ngx-bootstrap/#/rating#rating-basic) in the docs, you can see the issue with the aforementioned `span` element.

**Expected behavior**
The element should have an `aria-label` attribute on it. Ideally, this would be an input so that developers can add the most informative label for their use case.

### PR DESCRIPTION
**Pull request by IraErshova**
Add aria label attribute

Close #5579

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated tests.
 - [x] added/updated API documentation.
 - [x] added/updated demos.

### CODE REVIEW

**Review by daniloff200 — DISMISSED** [review]

File: src/rating/rating.component.ts [hunk 1]
```diff
@@ -30,6 +30,8 @@ export class RatingComponent implements ControlValueAccessor, OnInit {
   @Input() readonly: boolean;
   /** array of icons titles, default: (["one", "two", "three", "four", "five"]) */
   @Input() titles: string[];
+  /** custom aria label, default "rating" */
+  @Input() ariaLabel = 'rating';
```
Comment (valorkin) [inline src/rating/rating.component.ts]: please set it via config

**Review by valorkin — CHANGES_REQUESTED** [review]
todo: add config to rating component

File: src/rating/rating.component.ts (hunk 1 above)
Comment (IraErshova) [reply src/rating/rating.component.ts]: @valorkin I've added **Config** functionality, could you please re-check my PR?

**Review by valorkin — APPROVED** [review]
Looks great to me

**Comment by dmitry-zhemchugov** [comment]
Tested, Ready to merge

**Pull request merged**